chromium_src/third_party/blink/renderer/core/page/chrome_client_impl.cc

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "src/third_party/blink/renderer/core/page/chrome_client_impl.cc"
+
+#include "brave/third_party/blink/renderer/core/brave_session_cache.h"
+#include "third_party/blink/public/common/features.h"
+
+namespace blink {
+
+const display::ScreenInfos& ChromeClientImpl::BraveGetScreenInfos(
+    LocalFrame& frame) const {
```

Comment:
missing header for LocalFrame https://google.github.io/styleguide/cppguide.html#Include_What_You_Use

Reply:
Fixed